tools/docker/Dockerfile.ubuntu-build

```diff
@@ -0,0 +1,59 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:20.04 as builder
```

Comment:
1. what's a AIO?
2. Why do you prefer building dev env here instead of reusing the already built one : ghcr.io/romange/ubuntu-dev:20 ?

Reply:
This makes it more self-contained and comprehensive for the end-user. It's just an apt-get install, not special tooling :) Also, this way, the outputted container is more up2date on patches (and I feel like, Trivy imagine scanning should also be incorporated here)

Additionally, a special case which comes to my mind: I had a client with an air-gapped environment, where I would've been able to bring in a container image like ubuntu:20.04 and this Git repo as source code, but not neccessarily your dev container.

But honestly, I haven't investigated your dev container too much. Just couldn't find it's origin with 2 clicks and was relying on the instructions in the existing release workflow. Please just let me know your preference, and I'll change it accordingly :)

Reply:
it's fine, just wanted to understand your rationale.